docker-compose.yml

```diff
@@ -498,7 +498,7 @@ services:
     #     $ docker-compose pull python-manylinux1
     #   an then run:
     #     $ docker-compose run -e PYTHON_VERSION=3.7 python-manylinux1
-    image: quay.io/ursa-labs/arrow_manylinux1_x86_64_base:latest
+    image: ursalab/arrow_manylinux1_x86_64_base:0.14.1-static-zlib
```

Comment:
You should switch back to the "latest" label before merging this PR.

Reply:
The tag we're using is kinda inrelevant, and we can fall back to previous versions/tags. 
Additionally I don't need to re-push after it gets merged - so I'd rather keep it this way.

Reply:
It's relevant because it might break existing scripts, bash history, the READMEs in python/manylinux...

Reply:
The scripts and the documentation should use the `docker-compose run python-manylinux1` syntax, not the `docker run ursalab/arrow_manylinux1_x86_64_base:0.14.1-static-zlib` command.

Reply:
I'll check the documentation.

Reply:
The docker command is still necessary for rebuilding images.

Reply:
Also, the fact that "latest" is outdated isn't very intuitive...

Reply:
Updated the documentation.

Reply:
Ideally every docker image should have an explicit version, and the latest tag just should be a reference (symlink).

Reply:
Always updating the latest tag is a bad practice, this is true for the ursabot images as well (I'm working on a solution though).

Reply:
Hmm... how do you do a symlink? And is that automated?

Reply:
`docker tag python:3:7 ursalab/python:the-latest-python-image`